### DISCUSSION TITLE
Ordering by foreign table column not working

### DISCUSSION THREAD

**Discussion by mbalslow** [supabase/supabase#4255] [Questions]
I do not get the expected output when I attempt to order on foreign table columns. It seems to have no affect on the output.

Here is a very simple example of selecting all trips + the name of the related tour (many trips to one tour). I would expect based on this query that I get the trips ordered by the name of the related tour.

However, as you can see, that is not the case. It is the same whether I order ASC or DESC. Or if I use the new “!inner”.

I feel like this has worked before, so I am not sure if this is a new issue or not.

Am I doing something wrong, does it not work correctly, or something third?

![image](https://user-images.githubusercontent.com/49169557/144519293-4343955e-879b-416a-b22d-535826a09540.png)

**Comment by steve-chavez** [comment]
Yes, this is a pending enhancement: https://github.com/supabase/postgrest-js/issues/198

Right now `tour.order` only orders the `tour` table rows - which is not noticeable on a many-to-one relationship but would be on a one-to-many.

For now you could use a view or function for this.
